src/app/chart/map/map.component.ts

```diff
@@ -585,6 +613,31 @@ export class MapComponent extends SubscriptionManager {
     chart.seriesContainer.events.on('dragstop', () => {
       this.dragEndSubject.next(true);
     });
+
+    chartGlobe.events.on('ready', () => {
+      this.hideGlobe();
+
+      chartGlobe.deltaLongitude = -45;
+      chartGlobe.projection = new am4maps.projections.Orthographic();
+      chartGlobe.series.push(new am4maps.MapPolygonSeries()).useGeodata = true;
+      chartGlobe.seriesContainer.resizable = false;
+      chartGlobe.seriesContainer.draggable = false;
+
+      chartGlobe.animate({ property: 'deltaLongitude', to: 100050 }, 20000000);
+
+      setTimeout(() => {
+        chartGlobe.geodata = am4geodata_worldHigh;
+        const label = chartGlobe.chartAndLegendContainer.createChild(
+          am4core.Label
+        );
+        label.text = 'location unknown';
```

Comment:
my suggestion for the label, instead of `location unknown` use `europe`

Reply:
"Europe" is the misnomer currently used for the miscellaneous category - we actually don't know that this data is from Europe.